docker/files/webapp.conf

```diff
@@ -0,0 +1,8 @@
+server {
```

Comment:
wagnapp.tgz is pretty much just the result of running: wagn new wagnapp.

Note that you can't do: wagn new wagn (re-uses the Wagn constant)
The error is not right, it tries to throw an exception, but gets an undefined constant on the exception class.  That might be a leftover from the gem split.